test/Microsoft.AspNet.Razor.Test/Parser/CSharp/CSharpErrorTest.cs

```diff
@@ -28,6 +28,21 @@ public void ParseBlockHandlesQuotesAfterTransition()
         }
 
         [Fact]
+        public void ParseBlockWithHelperDirectiveProducesError()
+        {
+            ParseBlockTest("@helper { }",
+                           new ExpressionBlock(
```

Comment:
- `@helper someHelper {  }`
- nit: Use the new spacing format (aka 1 indent in).